frontend/src/components/stages/chat_editor.ts

```diff
@@ -39,17 +40,17 @@ export class ChatEditor extends MobxLitElement {
 
     // Check if experimenter's API key exists
     let apiCheck;
-    if (!this.authService.experimenterData?.apiKeys.geminiKey) {
+    if (!checkApiKeyExists(this.authService.experimenterData)) {
       apiCheck = html`
         <div class="warning">
-          <b>Note:</b> In order for LLM calls to work, you must add your Gemini
-          API key under Settings.
+          <b>Note:</b> In order for LLM calls to work, you must add your Gemini API key 
```

Comment:
Let's make this more generic: "you must add an API key or server configuration under Experimenter Settings." (Here and in chat_panel.ts)

Reply:
Fixed!